securedrop/sass/_base.sass

```diff
@@ -20,10 +20,14 @@
 
   h1, h2, h3
     font-weight: bold
+
+  h2, h3
     text-align: center
 
   h1
+    text-align: left
     font-size: 30px
+    color: #004080
```

Comment:
Actually the bigger problem is that most places we use `h2`, we should be using `h1`, and the only places we use `h1`, should definitely be `h2`. Ha. I'll make some more changes.